src/main/java/org/codehaus/mojo/taglist/TaglistReportRenderer.java

```diff
@@ -76,114 +71,82 @@ public class ReportGenerator {
      * @param report     the TagListReport object used in this build.
      * @param tagReports a collection of tagReports to output.
      */
-    public ReportGenerator(TagListReport report, Collection<TagReport> tagReports) {
-        sortedTagReports = new TreeSet<>(tagReports);
-        this.bundle = report.getBundle();
-        this.sink = report.getSink();
+    public TaglistReportRenderer(TagListReport report, Collection<TagReport> tagReports) {
+        super(report.getSink());
+        this.sortedTagReports = new TreeSet<>(tagReports);
         this.siteOutputDirectory = report.getReportOutputDirectory();
```

Comment:
The canonical name is `reportOutputDirectory` see `org.apache.maven.plugin.plugin.report.GoalRenderer`